template/assets/use_case/README.md.hbs

```diff
@@ -1,3 +1,5 @@
+![Aqora badge](https://img.shields.io/badge/aqora-4328e5?logo=data%3Aimage%2Fpng%3Bbase64%2CiVBORw0KGgoAAAANSUhEUgAAAA4AAAAOCAYAAAAfSC3RAAAACXBIWXMAAAsTAAALEwEAmpwYAAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAAIWSURBVHgBjZI%2FbBJxFMe%2F7%2F5x3BE9D2waCeZiEwNxECcbHSwmGt3qZkwcnB2Q0cFAR6d2Mk6KgcHNQSeHwmJaJ06XYmICprEq1PbaUk7ugJ93p1ToYPpb3svv5fve5%2F2%2BP8IRjqqdS4MGiwDTQFx1f2stx4%2BKip7Mi3KsKIVj87Kq7zj2z%2FqoJoajK15IAzTtxVlRiREFIi2ZB6Hg57yoIHEpC0k7lTGL16oRLTk3JFQmGcjkgkA0H4gEBWfSWShyHFIoet%2B%2FEyCbh9GJ2B%2Bhx24JvILU2SwiYhyCA4Q5feb6I3bRskyLGMuN6ZriEAsBqo9z3nhYkY%2FF0Q8BQznUjSQuvBseV9sshNzbArU0zdAAWbOsejOYOmpzY7ZiSJL%2BhMJKQoqeXh%2BqkuOogCuj48h4tlyg5Qncw%2Fz3brKrjoLbfRlTfQWwQ46w6zRmHM7edn5Zd8xi5t9EH8MVVUN0B5aPcvcWmwKHxX2%2Bc7Jtm5ddsaf4K2z9WEXrYznT2axXOVlLGg7JNdYf1BxCI6ynHpReUYvv4aVrfU1wdk8ResDel1V8r5XBBsj7wzie8NyLxgE7Q94nKL6h17B3Pglet%2B2N91hfK0%2BsxLEx0V9rgtfzM9feXdpsr6DRLI2V8SLw3PtmJ7xN58Z92rPqC37yeaPU7Pa%2FffDq0wSyGKPHXav%2B9OBxVD215HW64k1vSp7ZI6%2F%2Bd34DKO3EcR6BAbMAAAAASUVORK5CYII%3D&labelColor=eceafc&link=https%3A%2F%2Faqora.io%2Fcompetitions%2F{{competition}}%2F)
```

Comment:
I'm confused because a use case wouldn't have a "score" so what's the point of this badge. I mean we can have it here if aqora test is passing but it would work differently I believe